uwblueprint/sistering
# Change Sistering logo to use svg in volunteer navbar

## Ticket link

Closes #125 



## Implementation description
* Changed Sistering logo to the svg format



## Steps to test
1. Go to `/volunteer/shifts` or `/volunteer/postings` and check that the logo in the volunteer navbar is an svg



## What should reviewers focus on?
* Logo and spacing look good


## Checklist
